CHANGELOG.md

```diff
@@ -15,7 +17,8 @@
 
 ### Internal
 
-* Factor out internal interface ManagedObject
+* Factor out internal interface ManagedObject.
+* Use Object Store to do table initialization.
```

Comment:
The information about migration errors is not included in this PR. This will only contain the detailed information about the schema initialization issues (RealmModule issues) such as you defined ad `RealmObjectB` as a field of `RealmObjectA`, But `RealmObjectB` is not part of the `RealmModule`.